api_spec.yaml

```diff
@@ -86,15 +86,17 @@ paths:
                         $ref: '#/components/schemas/statusOK'
                       authorizationCode:
                         type: string
-                      timeCreated:
+                      timeCreatedMs:
                         $ref: '#/components/schemas/timestampMs'
-                      timeExpires:
+                      timeExpiresMs:
                         $ref: '#/components/schemas/timestampMs'
                   - type: object
                     properties:
```

Comment:
flow diagram changes:
- do not implicitly add openid to scopes if the scopes array is empty. We should instead throw an error, but this is not yet finalised. Depends on https://github.com/supertokens/docs/pull/664#discussion_r1218187806

Reply:
for now i made it throw 400 error if scopes are empty in the flow diagram

Reply:
Will accommodate, if there are any changes in adr later on.